### PR TITLE
Checklist Pre-v4.2

### PR DESCRIPTION
**Pull request by ThunderSon**
- Takes care of INFO-09 duplicates
- Adds INPV-19 (SSRF)

This is the final list for the IDs for the release.

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]
Thanks!

**Comment by ThunderSon** [comment]
I am considering updating the testing checklist md file to contain the columns `Status` which contains what this test's result was, and a `Notes` column.
Then I will remove the table from the Reporting, and instruct in there to attach the checklist as an additional document. The report shouldn't contain all tests conducted, but the tests that were relevant and with detailed findings.

Is this a sane approach? What do you think?

File: checklist/Testing_Checklist.md [hunk 1]
```diff
@@ -2,110 +2,116 @@
 
 The following is the list of controls to test during the assessment:
```
Comment (kingthorin) [inline checklist/Testing_Checklist.md:3]:
```suggestion
The following is the list of items to test during the assessment:
```
It's not a list of controls

**Comment by ThunderSon** [comment]
This is ready to be merged

**Review by kingthorin — APPROVED** [review]
Thanks!

**Pull request merged**